Assets/Scripts/Combat/TargetSelector.cs

```diff
@@ -26,13 +26,16 @@ public TargetSchema(int numberOfTargets, CombatantType combatantType, SelectorTy
 public class TargetSelector : MonoBehaviour
 {
     public CombatSystem CombatSystem;
+    private MouseSelect MouseSelector;
 
     private GameObject[] CurrentTargetedCombatants;
     private Ability CallingAbility;
     private TargetSchema CurrentTargetSchema;
 
     public void Start()
     {
+        MouseSelector = GetComponent<MouseSelect>();
+        MouseSelector.enabled = false;
```

Comment:
Looked this up cuz I didn't know the diff until now : https://forum.unity.com/threads/gameobject-setactive-vs-enabled.195658/ 

I'm guessing you mean to deactivate the component here. Not part of the review, just sharing the info in case haha

Reply:
Yes, this script is attached to the same empty object